plugins/modules/docker_compose_v2_run.py

```diff
@@ -424,7 +424,7 @@ def main():
         needs_api_version=False,
         **argspec_ex
     )
-    sanitize_labels(client.module.params['labels'], 'labels', client)
+    sanitize_labels_list(client.module.params['labels'], 'labels', client)
```

Comment:
Thinking a bit more about this, I think we can simply remove this line here and be done with it. Ansible already ensured (due to `type=list` and `elements=str`) that all elements of `client.module.params['labels']` are strings.

Sorry for not noticing that earlier!

Reply:
Yes, I thought the same thing when I saw it.... I imagined that maybe it was left because of some kind of “redundancy”.

It still hurts me that in all the other components it is a dictionary and here, by whims of fate, it is a list of strings.

Reply:
It was an oversight by me when adding the module (#969). Unfortunately nobody commented on it so far (especially not before merging the PR).

Reply:
Oook! changes done :)